charts/gardener-extension-admission-calico/values.yaml

```diff
@@ -7,6 +7,7 @@ global:
     repository: europe-docker.pkg.dev/gardener-project/public/gardener/extensions/admission-calico
     tag: latest
     pullPolicy: IfNotPresent
+# priorityClassName: gardener-garden-system-400
```

Comment:
Any reason not to enable it by default but to require the component that installs the admission charts to explicitly pass it?
If there is no concrete reason, I would suggest to set it by default. Motivation: to have more sane defaults that work out of the box.